production/tanka/grafana-agent/smoke/crow/main.libsonnet

```diff
@@ -14,8 +14,9 @@ local volume = k.core.v1.volume;
     local this = self,
 
     _config+:: {
-      image: 'grafana/agent-crow:latest',
+      image: 'us.gcr.io/kubernetes-dev/grafana/agent-crow:dev.smoke-framework',
```

Comment:
once we get this merged, built and deployed from main I'll open another PR to update this to use the `main` tag instead of `dev.smoke-framework`